packages/react/src/components/TextArea/TextArea.js

```diff
@@ -24,40 +44,78 @@ const TextArea = ({
   invalidText,
   helperText,
   light,
+  charCount,
```

Comment:
Probably `useCharCount`? (To distinguish it from the count (length) itself)

Reply:
sure, renamed this for `TextInput` and `TextArea`